src/protocols/vnc/client.h

```diff
@@ -54,6 +54,11 @@
  */
 #define GUAC_VNC_CLIPBOARD_MAX_LENGTH 262144
 
+/**
+ * The amount of time in milli-seconds to wait before sending a ping to VNC server.
+ */
+#define GUAC_VNC_IDLE_TIMEOUT 10000
```

Comment:
This value should probably be a connection configuration option, not a hard-coded value.

Reply:
Well like the other frame buffer timeout. `GUAC_VNC_CONNECT_INTERVAL` and other values I think it will be a hardcoded value as this is a server timeout wait check. So like rest if the user wants he can edit in the source file. Otherwise this option isn't going to be used by anyone normally.

Reply:
I think this also may be mis-named - the name `GUAC_VNC_IDLE_TIMEOUT` implies that this is the time that the connection is idle before it times out. Based on the comment, though, I think maybe something like `GUAC_VNC_PING_INTERVAL` would be more appropriate?